docs/netlify.toml

```diff
@@ -1,78 +1,78 @@
 [[redirects]]
-    from = "/guides/smart_contracts/writing_contracts/initializers"
-    to = "/guides/developer_guides/smart_contracts/writing_contracts/initializers"
+    from = "/build/smart_contracts/writing_contracts/initializers"
+    to = "/build/guides/smart_contracts/writing_contracts/initializers"
 
 [[redirects]]
     from = "/reference/aztecjs/accounts"
-    to = "/reference/developer_references/aztecjs/accounts"
+    to = "/build/reference/aztecjs/accounts"
 
 [[redirects]]
     from = "/getting_started"
-    to = "/guides/getting_started"
+    to = "/build/getting_started"
 
 [[redirects]]
-    from = "/tutorials/simple_dapp/*"
-    to = "/tutorials/codealong/simple_dapp"
+    from = "/build/tutorials/simple_dapp/*"
```

Comment:
this `/build/` in the path is new, so should the `from` remain the same, and the `to` is pointed to the `build/` path?

And I think we need to add redirects from guides to build like
```
from = "/guides/..."
to = "/build/..."
```